silq/pulses/pulse_types.py

```diff
@@ -667,6 +667,121 @@ def get_voltage(self, t: Union[float, Sequence]) -> Union[float, np.ndarray]:
         return waveform
 
 
+class MultiSinePulse(Pulse):
+    """MultiSinusoidal pulse
+
+    Parameters:
+        name: Pulse name
+        frequencies: list of Pulse frequencies
+        phases: list of Pulse phases
+        amplitudes: list of Pulse amplitudes
+        power: Pulse power
+        offset: amplitude offset (0 by default)
+        frequency_sideband: Mixer sideband frequency (off by default)
+        sideband_mode: Sideband frequency to apply. This feature must
```

Comment:
frequency_sideband and sideband_mode have nearly identical documentation. What do they exactly correspond to, and how do they differ?

Reply:
Honestly, I have never used those and don't really understand how they work. So I just copied from other pulses since it is required for the microwave interface. Can discuss in the office...

Reply:
yep I see it's the same issue in other pulses. We'll discuss later